diff -u -r ./oscats-0.6-orig/src/liboscats/algorithms/closest_diff.c ./oscats-0.6/src/liboscats/algorithms/closest_diff.c
--- ./oscats-0.6-orig/src/liboscats/algorithms/closest_diff.c	2011-10-26 00:23:34.000000000 +0100
+++ ./oscats-0.6/src/liboscats/algorithms/closest_diff.c	2012-09-02 00:29:07.000000000 +0100
@@ -175,7 +175,7 @@
   return oscats_model_distance(model, theta, e->covariates);
 }
 
-static gint select (OscatsTest *test, OscatsExaminee *e,
+static gint select2 (OscatsTest *test, OscatsExaminee *e,
                     GBitArray *eligible, gpointer alg_data)
 {
   return oscats_alg_chooser_choose(OSCATS_ALG_CLOSEST_DIFF(alg_data)->chooser,
@@ -194,7 +194,7 @@
   OscatsAlgClosestDiff *self = OSCATS_ALG_CLOSEST_DIFF(alg_data);
   self->chooser->bank = g_object_ref(test->itembank);
   self->chooser->criterion = (OscatsAlgChooserCriterion)criterion;
-  g_signal_connect_data(test, "select", G_CALLBACK(select),
+  g_signal_connect_data(test, "select", G_CALLBACK(select2),
                         alg_data, oscats_algorithm_closure_finalize, 0);
 }
                    
diff -u -r ./oscats-0.6-orig/src/liboscats/algorithms/max_fisher.c ./oscats-0.6/src/liboscats/algorithms/max_fisher.c
--- ./oscats-0.6-orig/src/liboscats/algorithms/max_fisher.c	2011-10-26 00:24:19.000000000 +0100
+++ ./oscats-0.6/src/liboscats/algorithms/max_fisher.c	2012-09-02 00:28:33.000000000 +0100
@@ -265,7 +265,7 @@
     // max det[sum I_j(theta)] <==> min -det[sum I_j(theta)]
 }
 
-static gint select (OscatsTest *test, OscatsExaminee *e,
+static gint select2 (OscatsTest *test, OscatsExaminee *e,
                     GBitArray *eligible, gpointer alg_data)
 {
   OscatsAlgMaxFisher *self = OSCATS_ALG_MAX_FISHER(alg_data);
@@ -298,7 +298,7 @@
 
   g_signal_connect_data(test, "initialize", G_CALLBACK(initialize),
                         alg_data, oscats_algorithm_closure_finalize, 0);
-  g_signal_connect_data(test, "select", G_CALLBACK(select),
+  g_signal_connect_data(test, "select", G_CALLBACK(select2),
                         alg_data, oscats_algorithm_closure_finalize, 0);
   g_object_ref(alg_data);
 }
diff -u -r ./oscats-0.6-orig/src/liboscats/algorithms/max_kl.c ./oscats-0.6/src/liboscats/algorithms/max_kl.c
--- ./oscats-0.6-orig/src/liboscats/algorithms/max_kl.c	2011-10-31 21:43:53.000000000 +0000
+++ ./oscats-0.6/src/liboscats/algorithms/max_kl.c	2012-09-02 00:29:30.000000000 +0100
@@ -623,7 +623,7 @@
                                  alg_data);
 }
 
-static gint select (OscatsTest *test, OscatsExaminee *e,
+static gint select2 (OscatsTest *test, OscatsExaminee *e,
                     GBitArray *eligible, gpointer alg_data)
 {
   OscatsAlgMaxKl *self = OSCATS_ALG_MAX_KL(alg_data);
@@ -661,7 +661,7 @@
 
   g_signal_connect_data(test, "initialize", G_CALLBACK(initialize),
                         alg_data, oscats_algorithm_closure_finalize, 0);
-  g_signal_connect_data(test, "select", G_CALLBACK(select),
+  g_signal_connect_data(test, "select", G_CALLBACK(select2),
                         alg_data, oscats_algorithm_closure_finalize, 0);
   g_object_ref(alg_data);
 }
diff -u -r ./oscats-0.6-orig/src/liboscats/algorithms/pick_rand.c ./oscats-0.6/src/liboscats/algorithms/pick_rand.c
--- ./oscats-0.6-orig/src/liboscats/algorithms/pick_rand.c	2010-08-23 01:37:33.000000000 +0100
+++ ./oscats-0.6/src/liboscats/algorithms/pick_rand.c	2012-09-02 00:27:54.000000000 +0100
@@ -35,7 +35,7 @@
 {
 }
 
-static gint select (OscatsTest *test, OscatsExaminee *e,
+static gint select2 (OscatsTest *test, OscatsExaminee *e,
                     GBitArray *eligible, gpointer alg_data)
 {
   guint i, item = 0;
@@ -55,7 +55,7 @@
  */
 static void alg_register (OscatsAlgorithm *alg_data, OscatsTest *test)
 {
-  g_signal_connect_data(test, "select", G_CALLBACK(select),
+  g_signal_connect_data(test, "select", G_CALLBACK(select2),
                         alg_data, oscats_algorithm_closure_finalize, 0);
 }
                    
